internal/errors/errors.go

```diff
@@ -0,0 +1,96 @@
+// @license
+// Copyright 2025 Dynatrace LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package errors
+
+import (
+	"fmt"
+)
+
+// ErrorClient represents error wrapper that wraps errors happening on the clients
+type ErrorClient struct {
+	// Wrapped contained the raw error returned from the client
+	Wrapped error
+
+	// Identifier being not empty the error message will contain the value set
+	Identifier string
+
+	// Operation is the HTTP method that was used by the client
+	Operation string
+
+	// Resource is a string of the resource
```

Comment:
Please add here what a resource is. 
`Resource is a string of the resource` does not tell me anything new that `Resource string` didn't tell me before 😅 

Same with `Identifier`. 

`Operation` -> is this the method (`GET/POST/PUT`) or something more rest friendly aka `List/Update/Create`)?